Examples/NewModuleTemplate/plasterManifest.xml

```diff
@@ -53,6 +53,10 @@
         </parameter>
     </parameters>
     <content>
+        <message>
+Scaffold a PowerShell Module with the files required to run Pester tests, build with PSake and publish to the PSGallery.
+
```

Comment:
That is correct.  The XmlDocument parser isn't doing that.  That is what this line does:
```powershell
$trimmedText = $text -replace '^[ \t]*\n','' -replace '\n[ \t]*$','' 
```
Where this can bite you is if you do this:
```<message>`nSome message`n</message>```
We strip off the beginning and ending newlines.  So you'd have to do this to get the desired effect:
```<message>`n`nSome message`n`n</message>```
or
```xml
<message>

Some message

</message>
```